### ISSUE TITLE
GHCI Error in Atom GUI + Terminal opening with node error when starting haskell-debug

### ISSUE THREAD

**Issue by tschacht** [atom-haskell/haskell-debug#14]
Hi,

I'm trying to debug this Haskell application: https://github.com/neo4j-examples/neo4j-movies-haskell-bolt

and get the following error in Atom GUI:

```
GHCI Error
Configuring GHCi with the following packages: hasbolt-sample-app
Using main module: 1. Package `hasbolt-sample-app' component exe:hasbolt-sample-app-exe with main-is file: /Users/ts/work/haskell/neo4j-movies-haskell-bolt/app/Main.hs
```

also a Terminal opens up with this content:

```
node /Users/ts/.atom/packages/haskell-debug/lib/TerminalEcho.js
ts@petunia ~ 0% node /Users/ts/.atom/packages/haskell-debug/lib/TerminalEcho.js
/Users/ts/.atom/packages/haskell-debug/lib/TerminalEcho.js:3
const net = require("net");
^^^^^
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```
I have no idea whats wrong here. Any hints?

Thanks,
BR

**Comment by tschacht** [comment]
I'm using Atom 1.19.2 x64 on OSX 10.12.6 with haskell-debug 0.3.2 and have all other "atom-haskell" packages installed with current versions.

**Comment by tschacht** [comment]
Also this error seems to be specific to this Haskell application as I can use haskell-debug with other applications without problems.

**Comment by lierdakil** [comment]
You can ignore "GHCI Error", it's just something `stack` spits out on stderr when running `stack repl`. Node error in terminal is my fault. Terribly sorry about that, working on a fix right now.

**Comment by lierdakil** [comment]
Just released v0.3.3, which should fix node error (at least limited testing I did shows that it should). Please let me know if it works for you.

**Comment by tschacht** [comment]
The node error is gone and I see a working GHCI terminal. :-) Thanks for the quick fix. 👍🏼
